app/assess/models/flag.py

```diff
@@ -8,18 +8,14 @@ class FlagType(Enum):
     FLAGGED = 0
     STOPPED = 1
     QA_COMPLETED = 2
-
-
-class ResolutionType(Enum):
-    QUERY_RESOLVED = 0
-    STOP_ASSESSMENT = 1
+    RESOLVED = 3
```

Comment:
Does this need a corresponding database migration?

Reply:
yes it's here: https://github.com/communitiesuk/funding-service-design-assessment-store/pull/98/files#diff-ec0465a8bc7a7def015e2569574477919c9d91d1971736f1fc233249e023f22c

Reply:
As that is not merged yet, shall we make this draft until that is merged, because this won't work until then?

Reply:
yes sure 👍